recipes/metrics-server/metrics-server-deployment.yaml

```diff
@@ -36,7 +36,7 @@ spec:
         # - --metric-resolution=10s
         - --kubelet-insecure-tls
         - --kubelet-preferred-address-types=InternalIP
-        image: k8s.gcr.io/metrics-server-amd64:v0.3.1
+        image: k8s.gcr.io/metrics-server-arm:v0.3.1
```

Comment:
We probably shouldn't change the default from amd64 to arm, the right thing to do is get metrics-server to use a multi-arch image.

Reply:
I've checked, and I couldn't find metric server multi-arch image, However i found a related opened pull request  [PR #182](https://github.com/kubernetes-incubator/metrics-server/pull/182).

Reply:
can you switch this to `rancher/metrics-server:v0.3.1` as we package it as multi-arch and we can use that until upstream adds in multi-arch.

Reply:
@deniseschannon 

I tried to test rancher metrics-server multi-arch image you mentioned on the same raspberry pi, but i got an error stating that:
`no matching manifest for unknown in the manifest list entries`

The raspberry pi architecture is **armv7l**

Reply:
Thanks for trying it out @meldafrawi! Is it possible for us to add armv7 support to rancher/metrics-server @deniseschannon?

Reply:
@erikwilson, You're welcome.

Reply:
@erikwilson we'd have to sync up internally that created this image and create PRs to be able to do that. can you reach out to see who created it?

Reply:
/ping @loganhz

Reply:
We don’t have a arm7 drone box.
So our CI only has amd64 build and arm64 build.

We need to talk to Darren and Bill about it. @erikwilson

Reply:
@loganhz we do have arm7.  k3s CI uses it.  You just need to put `arm` as the architecture in .drone.yaml.  It's technically a ARM64 VM but running a armv7 version of docker which pulls armv7 architecture images.

Reply:
Sorry. Just recalled. @cjellick helped to pushed the metrics server images manually to rancher dockerhub for the ARM64 Support. 
https://github.com/jianghang8421/publish-manifest/blob/master/publish-manifest-images.sh#L102-L112

So we don't have a CI build for rancher/metrics-server images currently. (Good to know we have arm7 build!)

We need someone to pull `gcr.io/google_containers/metrics-server-arm:v0.3.1`, retag and update the `rancher/metrics-server` manifest images manually.

Reply:
I'll look into this, @loganhz thanks

Reply:
Guys, I do Travis builds for multiple ARM architectures using QEMU,. Check out https://github.com/insightfulsystems/alpine-python/blob/master/.travis.yml